lib/core/versions/latest/Operation.ts

```diff
@@ -342,6 +342,10 @@ export default class Operation {
         throw new SidetreeError(ErrorCode.OperationUpdatePatchPublicKeyIdNotString);
       }
 
+      if (typeof publicKey.controller === 'string') {
```

Comment:
```suggestion
      if (typeof publicKey.controller !== undefined) {
```

Reply:
Lint is pointing out that the suggestion ```typeof publicKey.controller !== undefined``` is always true so I'm going to change it to check whether the 'controller' property is defined in that section at all or not.

Reply:
Yeah... that's what I mean, was a typo.